src/Plugin.php

```diff
@@ -199,22 +200,29 @@ public function onDependenciesChangedEvent()
      */
     private function loadInstalledPaths()
     {
-        if ($this->isPHPCodeSnifferInstalled() === true) {
-            $this->processExecutor->execute(
-                sprintf(
-                    'phpcs --config-show %s',
-                    self::PHPCS_CONFIG_KEY
-                ),
-                $output,
-                $this->composer->getConfig()->get('bin-dir')
-            );
+        if ($this->isPHPCodeSnifferInstalled() !== true) {
+            return;
+        }
+
+        $this->processExecutor->execute(
+            sprintf(
+                'phpcs --config-show %s',
+                self::PHPCS_CONFIG_KEY
+            ),
+            $output,
+            $this->composer->getConfig()->get('bin-dir')
+        );
 
-            $phpcsInstalledPaths = str_replace(self::PHPCS_CONFIG_KEY . ': ', '', $output);
-            $phpcsInstalledPaths = trim($phpcsInstalledPaths);
+        $regex = sprintf(self::PHPCS_CONFIG_REGEX, self::PHPCS_CONFIG_KEY);
+        if (preg_match($regex, $output, $match) !== 1) {
+            return;
```

Comment:
I would rather have the code that is supposed to be run for the `true` scenario onside the `if` body, rather than (ab)using an early return.